src/components/WebexMember/WebexMember.scss

```diff
@@ -12,6 +12,12 @@
     padding-left: 0.25rem;
   }
 
+  .member-subtitle {
```

Comment:
let's use `subtext`, like in the story, and no `member-` prefix, we are in the member component context anyway